src/tests/ftest/control/daos_admin_privileged.py

```diff
@@ -62,37 +64,39 @@ def test_daos_admin_format(self):
         try:
             self.server_managers[0].prepare_storage(user, True, False)
         except ServerFailed as error:
-            self.fail("Failed preparing SCM as user {}: {}".format(user, error))
+            self.fail(
+                "(1)Failed preparing SCM as user {}: {}".format(user, error))
```

Comment:
Minor - we now allow lines up to 100 characters, so these 2 lines could fit in 1 line.  This may apply to other edited lines.
```suggestion
            self.fail("(1)Failed preparing SCM as user {}: {}".format(user, error))
```

Reply:
Thanks for the comment.